TRANSLATION.md

```diff
@@ -0,0 +1,144 @@
+Translating Kibana
+===
+
+Background
+---
+
+Please see [kibana#6515](https://github.com/elastic/kibana/issues/6515)
+for background discussion on the Kibana translation work.
+
+Prerequisites
+---
+
+Kibana must be installed and operational, see README.md
+
+Audience
+---
+
+There are two audiences for this document:
+- those that want to contribute translations to Kibana
+- those that want to enable translation of Kibana
+
+Contributing Translations
+---
+
+For this example, we will demonstrate translation into Maltese (Language code `mt`).
+
+- Check out the `kibana` source code with `git`,
```

Comment:
This should instruct the user to fork the Kibana repo rather than just checking it out.